.gitallowed

```diff
@@ -0,0 +1,3 @@
+MyPASS=${2-"password"}
+Using a password on the command line interface can be insecure
+'password' => 'password'
```

Comment:
**New file**

Added to prevent git secrets blocking commits for these false positives